app/(api)/_datalib/collections/findItem.ts

```diff
@@ -0,0 +1,41 @@
+import { getClient } from '@utils/mongodb/mongoClient.mjs';
+import { NotFoundError } from '@utils/response/Errors';
+
+export async function findCollectionItems(collection: never, query = null) {
+  try {
+    const client = await getClient();
+    const db = client.db;
+    const reqCollection = await db.getCollection(collection);
+    let reqDocument;
+
+    if (query === null) {
+      reqDocument = await reqCollection.find().toArray();
+    } else {
+      reqDocument = await reqCollection.find(query).toArray();
+    }
```

Comment:
This can be reduced a little. If you give query a default value of {}, then you can always just pass in query and you don't have to check for a null query. Simplifies the code a little since you're using a default value for query anyways.

Reply:
Hey Austin, I made the requested changes let me know if it's all good?.